PluginsAndFeatures/azure-toolkit-for-intellij/rider/src/org/jetbrains/plugins/azure/functions/AzureCoreToolsMissingNupkgInstaller.kt

```diff
@@ -81,10 +90,16 @@ class AzureCoreToolsMissingNupkgInstaller : StartupActivity {
                 for (event in events) {
                     val file = event.file ?: continue
 
-                    if (shouldProcess(projectFileIndex, file) &&
-                            hasKnownFileSuffix(file) && // Only process certain file types
-                            event.requestor !is FileDocumentManagerImpl && // Don't process when typing in document
-                            file.length > 0) {
+                    if (hasKnownFileSuffix(file) &&
+                            existsInCurrentProject(projectFileIndex, file) &&
+                            isNewOrMarkedForProcessing(file, event)) {
+
+                        // First pass(es), no content will be in the file.
+                        // If that is the case, mark the file for processing on later changes.
+                        if (file.length < 25 || event is VFileCreateEvent) {
```

Comment:
Why do we check for `length < 25` here, not just `!= 0`?

Reply:
When creating a new file from our templates, we get these VFS events:
* Create - length == 0
* Update - length == 3 (have seen 10 as well, chose 25)
* Update - length == actual template length

Reply:
I would add some explanation in code :)